### PR TITLE
readme: link tracking issue and matrix room

### PR DESCRIPTION
**Pull request by Vagahbond**
I wanted to try Cosmic and ended up here after googling my way to the tracking issue. 

Thanks for taking the time to make this flake!

I thought this readme could lead back to the tracking issue for people to easily find what is going on and test Cosmic on NixOS. 

Same for the matrix room.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # NixOS COSMIC
 
-Nix package set and NixOS module for using COSMIC from NixOS. This is a temporary repository for testing COSMIC on NixOS as it is developed. When COSMIC gets more stable and it is fully working on NixOS, these packages and module are intended to be merged upstream into nixpkgs.
+Nix package set and NixOS module for using COSMIC from NixOS. This is a temporary repository for testing COSMIC on NixOS as it is developed. When COSMIC gets more stable and it is fully working on NixOS, these packages and module are intended to be merged upstream into nixpkgs(track progress [here](https://github.com/NixOS/nixpkgs/issues/259641).
```
Comment (lilyinstarlight) [inline readme.md:3]: This issue doesn't really track progress of this repo. So it should probably have different wording (e.g. "issue to be updated when upstreaming")

File: readme.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
+Dedicated development matrix room: https://matrix.to/#/#cosmic:nixos.org
```
Comment (lilyinstarlight) [inline readme.md:5]:
```suggestion
Dedicated development matrix room: <https://matrix.to/#/#cosmic:nixos.org>
```

File: readme.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 # NixOS COSMIC
 
-Nix package set and NixOS module for using COSMIC from NixOS. This is a temporary repository for testing COSMIC on NixOS as it is developed. When COSMIC gets more stable and it is fully working on NixOS, these packages and module are intended to be merged upstream into nixpkgs.
+Nix package set and NixOS module for using COSMIC from NixOS. This is a temporary repository for testing COSMIC on NixOS as it is developed. When COSMIC gets more stable and it is fully working on NixOS, these packages and module are intended to be merged upstream into nixpkgs([issue to be updated when upstreaming](https://github.com/NixOS/nixpkgs/issues/259641).
```
Comment (lilyinstarlight) [inline readme.md:3]:
```suggestion
Nix package set and NixOS module for using COSMIC from NixOS. This is a temporary repository for testing COSMIC on NixOS as it is developed. When COSMIC gets more stable and it is fully working on NixOS, these packages and module are intended to be merged upstream into nixpkgs ([issue to be updated when upstreaming](https://github.com/NixOS/nixpkgs/issues/259641)).
```

Comment (Vagahbond) [reply readme.md:3]: Oops sorry

**Review by lilyinstarlight — APPROVED** [review]
Looks good, thank you again!

**Pull request merged**